src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -53,17 +53,18 @@ const DuplicateVolumeForm: FC<Props> = ({ volume, close }) => {
   });
 
   const notifySuccess = (volumeName: string, project: string, pool: string) => {
+    const url = `/ui/project/${project}/storage/pool/${pool}/volumes/custom/${volumeName}/configuration`;
     const message = (
       <>
-        Created volume <strong>{volumeName}</strong>.
+        Created volume{" "}
+        <ResourceLink type="volume" value={volumeName} to={url} />.
```

Comment:
Here we also link the config tab, where the target url for the chip should be the overview page.

Reply:
resolved with the previous comment